logsearchapi/main.go

```diff
@@ -31,29 +31,22 @@ import (
 	"github.com/minio/operator/logsearchapi/server"
 )
 
-const (
-	pgConnStrEnv      = "LOGSEARCH_PG_CONN_STR"
-	auditAuthTokenEnv = "LOGSEARCH_AUDIT_AUTH_TOKEN"
-	queryAuthTokenEnv = "LOGSEARCH_QUERY_AUTH_TOKEN"
-	diskCapacityEnv   = "LOGSEARCH_DISK_CAPACITY_GB"
-)
-
 func loadEnv() (*server.LogSearch, error) {
```

Comment:
This function can be moved as well into the `server` package such as 

```
slog, err := server.LoadFromEnv()
```

Reply:
done